pkg/registry/core/service/storage/rest_test.go

```diff
@@ -1418,27 +963,13 @@ func TestDualStackServiceRegistryDeleteDryRun(t *testing.T) {
 
```

Comment:
I think we can merge the dualstack test into the normal test and replay on # of Cluster IPs in the original service.

Reply:
Similarly we can merge all DeleteXXXDryRun(...) tests into one test.

Reply:
Defer into the delayering process, if that's OK